packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -114,6 +115,8 @@ class IconLabelButton extends StatelessWidget {
 
   final String tooltip;
 
+  final EdgeInsetsGeometry tooltipPadding;
```

Comment:
why is this needed? Is there a reason we can't just give all tooltips a reasonable default?

Reply:
Multiline tooltips will look bad without specifying the padding. Single line tooltips have adequate spacing on the top and bottom, but if there is a new line char in the tooltip message, that top and bottom spacing disappears.